src/config-file.ts

```diff
@@ -12,6 +12,8 @@ import * as output from './output';
 const LEGACY_CONFIG_FILENAME = '.airc';
 const CONFIG_FILENAME = '.airc.json';
 
+const CHATS_SAVE_DIRECTORY = '/ai-chats';
```

Comment:
THis looks like it would be saving to absolute path `/ai-chats`. Change to `~/ai-chats`.

Reply:
fixed